mypy/stubgen.py

```diff
@@ -574,6 +575,16 @@ def get_qualified_name(o: Node) -> str:
         return '<ERROR>'
 
 
+def walk_packages(packages: List[str]):
+    for package_name in packages:
+        package = __import__(package_name)
```

Comment:
Oh, good point! I don't believe importing is strictly necessary here, since at this point I'm simply trying to walk the packages. I'll look into re-implementing this function without `__import__`